### ISSUE TITLE
"Haves"

### ISSUE THREAD

**Issue by darabos** [bjascob/LemmInflect#4]
```python
>>> import lemminflect as li
>>> li.getInflection('have', 'VBZ')
('haves',)
```

Shouldn't that be `has`? What am I doing wrong?

**Comment by bjascob** [comment]
I think you're correct.  Looks like this is a bug.
I just did a commit that fixes this (and a similar issue with 'do').  If you install the master from Github (not pypi) you'll get the updates.  Let me know if you see anything else.

**Comment by darabos** [comment]
Awesome, thanks!

**Comment by bjascob** [comment]
Issue closed with commit 51c3d96ffa2ea36bb1ccd956b2ebe2c96d73fe99 and release 0.2.1
